lib/puppet_x/templates/invoke_ps_command.erb

```diff
@@ -23,7 +215,10 @@ public class PuppetPSHost : PSHost
     public override Guid InstanceId { get { return _hostId; } }
     public override string Name { get { return "PuppetPSHost"; } }
     public override Version Version { get { return new Version(1, 0); } }
```

Comment:
The `PSHost` version indicates the version of the hosting application, not PowerShell. The `ConsoleHost` host and the `ISE` host share the same number as the version of PowerShell installed, but as @glennsarti points out `WinRM` and other hosts like `Chocolately` don't. The Host name and version are supposed to allow you to make the decision as a module author whether or not you support running on that host. I'd argue anyone specifying a specific host `name` and `version` know what they are doing enough to make the call.

The alternative is to mimic the ConsoleHost host name and version, which is misleading and may lead to unintended consequences.